ps_socialfollow.php

```diff
@@ -93,14 +92,19 @@ public function uninstall()
 
     public function getContent()
     {
+        $html = '';
         if (Tools::isSubmit('submitModule')) {
```

Comment:
Is this the correct way to override `postProcess` ? It does not look very clean, but I didn't find a better example in other modules.

Reply:
The `getContent()` old-fashioned way requires us to do this. I think indeed it's "good enough" for the current scope 👍